### PR TITLE
End-of-rgsoc-Blogpost

### PR DESCRIPTION
**Pull request by froilainhaeckse**
To be published september 21st!

### CODE REVIEW

File: blog/_posts/2016-09-21-Buffering.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Buffering
+layout: post
+created_at: 2016-09-21
+permalink: blog/2016-09-19-Buffering
+current: blog
+author: Thea Amanda Kupler and Marie-Luise Kochsiek
+twitter: @loadtocode
```
Comment (alicetragedy) [inline blog/_posts/2016-09-21-Buffering.md]: please remove the `@` symbol, as it breaks Jekyll and keeps it from building properly.

File: blog/_posts/2016-09-21-Buffering.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+categories:
+- student-posts
+---
+## <span class="color-red">...so the Rails Girls Summer of Code will come to an end!</span>
+
+Two months ago we blogged about our start - about us being [loaded to code](/blog/2016-07-21-start-of-loadtocode). Now, having some time passed, which seems to go faster day by day, there is this constant feeling of loading, loading knowledge to code or rather buffering - buffering all the knowledge we received those last months. We not only received great coding teaching but also valuable business knowledge. We reached out to the amazing SoundCloud-People surrounding us and mentioned that we are a little bit lost in our learning freedom and not having a structure set from top-down. We saw the problem, reached out and found solutions - creating our own structure. BOTTOM(S)-UP.
+
+![LoadtoCode KANBAN](/img/blog/2016/2016-09-21-KANBAN.gif) <font color="grey"><small><i>READY, STEADY, PUSH, DONE: Our KANBAN-Board in movement!</i></small></font><br>
```
Comment (alicetragedy) [inline blog/_posts/2016-09-21-Buffering.md]: add image credits to all of your images, as explained [here](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CONTRIBUTING.md#images).

File: blog/_posts/2016-09-21-Buffering.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: Buffering
+layout: post
+created_at: 2016-09-21
```
Comment (alicetragedy) [inline blog/_posts/2016-09-21-Buffering.md]: please change the format here to `Wed Sep 21 2016`

File: blog/_posts/2016-09-21-Buffering.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+permalink: blog/2016-09-19-Buffering
```
Comment (alicetragedy) [inline blog/_posts/2016-09-21-Buffering.md]: the permalink should have the correct date, so `blog/2016-09-21-buffering`

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Hey @theaamanda, great post! I've made a few comments on things that need to be changed before we can publish your blog post. Please ping me if you have any questions!

**Comment by froilainhaeckse** [comment]
@alicetragedy Hey Laura, please check if its okay like this now!

**Review by alicetragedy — APPROVED** [review]
perfect, this is good to go now. Thanks!

**Pull request merged**